build.gradle

```diff
@@ -32,11 +33,16 @@ subprojects {
     apply plugin: 'com.palantir.baseline-eclipse'
     apply plugin: 'com.palantir.baseline-findbugs'
     apply plugin: 'com.palantir.baseline-idea'
+    apply plugin: 'com.palantir.docker-test-runner'
     apply plugin: 'org.inferred.processors'
 
     repositories {
         jcenter()
     }
 
     sourceCompatibility = 1.7
+
+    if (System.env.CIRCLE_TEST_REPORTS) {
+        test.reports.junitXml.destination = new File(System.env.CIRCLE_TEST_REPORTS, it.getName())
```

Comment:
Not strictly necessary, but doing this populates a nicer over-all test summary on Circle.

If you take a look at https://circleci.com/gh/palantir/http-remoting/130, you'll see that under "Test Summary" it says: "Help us provide better insight around your tests and failures. Set up your test runner to output in JUnit-style XML, so we can..."

Compare to https://circleci.com/gh/palantir/http-remoting/131 (this PR), which properly says that 149 tests were run and provides a (very) brief summary.

See https://circleci.com/docs/test-metadata/#metadata-collection-in-custom-test-steps for details.